main/src/ui/contact_details/muc_config_form_provider.vala

```diff
@@ -63,12 +63,12 @@ public class MucConfigFormProvider : Plugins.ContactDetailsProvider, Object {
                     label = _("Occupants may change the subject");
                     break;
                 case "muc#roomconfig_whois":
-                    label = _("Discover real JIDs");
```

Comment:
This sounded like from an advertisement or a game :wink: I think it's not about active discovering, but being allowed to view something.

Also, "real" sounds like there are other JIDs, like fake ones. I think there are only JIDs and aliases, so I removed "real".